src/node_debug_options.cc

```diff
@@ -77,9 +77,11 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
   if (option_name == "--inspect") {
     enable_inspector = true;
-  } else if (option_name == "--inspect-brk") {
+  } else if (option_name == "--inspect-brk" || option_name == "--debug-brk") {
```

Comment:
@jasnell added that to first comment
I believe that @jkrems also requested the `--debug-brk` alone will fail, and that only the combo `--inspect --debug-brk` will be valid.